dbt_subprojects/tokens/models/tokens/tokens_v1_erc20.sql

```diff
@@ -5,26 +5,31 @@
         ,materialized = 'table'
         ,partition_by = ['blockchain']
         ,post_hook='{{ expose_spells(\'[
```

Comment:
We'll want to remove the post hook here as well and hide it from the data explorer